tests/patterns/drafting.mjs

```diff
@@ -127,7 +125,7 @@ export const testPatternDrafting = (design, Pattern, expect, models, patterns, l
     for (const sa of [0,10]) {
       it(`  - Drafting paperless non-detailed pattern for size-40 (${breasts ? 'with' : 'no'} breasts) sa: ${sa}`, () => {
         expect(
-          doesItDraft(
+          doesItDraftAndRender(
```

Comment:
This entire `settings` prop needs to go. It's there for god knows what reason, but has no effect.

Reply:
Note that this remark is about the lines below, not what you changed.

Reply:
Gone!